src/GitPrompt.ps1

```diff
@@ -128,11 +130,18 @@ if (Get-Module NuGet) {
 }
 
 function Write-Prompt($Object, $ForegroundColor, $BackgroundColor = -1) {
+    if ($GitPromptSettings.AnsiConsole) {
+        $e = [char]27 + "["
+        $f = Get-ForegroundVirtualTerminalSequence $ForegroundColor
+        $b = Get-BackgroundVirtualTerminalSequence $BackgroundColor
+        return "${f}${b}${Object}${e}0m"
```

Comment:
`$ForegroundColor` and `$BackgroundColor` are explicitly untyped, so in theory there's no reason `Get-ForegroundVirtualTerminalSequence` and `Get-BackgroundVirtualTerminalSequence` couldn't emit valid escape sequences given a variety of input types (`"#rrggbb"`, [`System.Drawing.Color`](https://msdn.microsoft.com/en-us/library/system.drawing.color.aspx), etc).

(Not that a `Color` class couldn't have equivalent support.)